source/docs/advanced/theme/analytics.md

```diff
@@ -0,0 +1,237 @@
+    debug: true,
+    defaultSettings: {},
+    // The list of integrations is defined here
+    integrations: [
+      {
+        // The name allows to know if the user allowed to use
+        // this tracking service or not
+        name: "google-analytics",
+        // Usually we always need to set it to true since GDPR
+        needConsent: true,
+        settings: {
+          // Settings needed by the integration
+          // The fact that it the key "Google Analytics" is
+          // defined by the integration itself
+          "Google Analytics": {
+            trackingId: "123456",
+            enhancedEcommerce: true
+          }
+        },
+        // integration that will add itself to the `analytics.js` lib
+        script: () =>
+          require("@segment/analytics.js-integration-google-analytics")
+      }
+      // You can add other integrations here. They are loaded asynchronously and
+      // won't impact the user's performance too much (as long as there are not
+      // too many).
+    ]
+  }
+};
+```
+
+Here we have used the Google Analytics integration. You can find the [list of existing integrations here](https://github.com/segment-integrations).
+
+<blockquote class="warning">
+However, please note that some integrations are not up to date. Even if you find one that matches your need, it might not work. However, looking into its code will let you know how to make it work, and you could eventually fork it to fix it.
+</blockquote>
+
+### GDPR consent
+
+If your integrations need consent of the user before running, you need to setup the `config/cookieServices.js` file. This file will let you define which cookies and trackings services are used within your application and will let the user chose which tracking service to allow.
+
+```js
+export default [
+  {
+    // Category of cookies to allow the user to accept all the integrations at once in a specific category
+    title: "Analytics",
+    description:
+      "These cookies allows us to measure the traffic on our contents and hence to improve them.",
+    services: [
+      {
+        // The name should be the same as mentionned in the `config/analytics.js` file
+        name: "google-analytics",
+        title: "Google Analytics",
+        // display all the cookies managed by Google Analytics
+        cookies: [
+          "_ga",
+          "_gat",
+          "_gid",
+          "__utma",
+          "__utmb",
+          "__utmc",
+          "__utmt",
+          "__utmz"
+        ],
+        description:
+          "Google Analytics cookies, from Google, are meant to gather statistics about visits.",
+        link: "https://support.google.com/analytics/answer/6004245"
+      }
+    ]
+  }
+];
+```
+
+### How to create a custom integration?
+
+<blockquote class="wip">
+Currently, the integrations creation is not documented by Segment.io. We plan to add information here to help you to:
+* define an integration that will add a specific pixel on a specific page
```

Comment:
The list is not rendered properly here.

We should use `<ul>` and `<li>`s in this case (I have not yet found a better solution).